fe/fe-core/src/main/java/com/starrocks/catalog/HiveTable.java

```diff
@@ -188,10 +187,55 @@ public HiveTableStats getTableStats() throws DdlException {
         return result;
     }
 
-    public void refreshTableCache() throws DdlException {
-        Catalog.getCurrentCatalog().getHiveRepository()
-                .refreshTableCache(resourceName, hiveDb, hiveTable, getPartitionColumns(),
-                        new ArrayList<>(nameToColumn.keySet()));
+    public void refreshTableCache(String dbName, String tableName) throws DdlException {
+        org.apache.hadoop.hive.metastore.api.Table table = Catalog.getCurrentCatalog().getHiveRepository()
```

Comment:
import org.apache.hadoop.hive.metastore.api.Table;

Reply:
org.apache.hadoop.hive.metastore.api.Table;  and com.starrocks.catalog.Table; cannot exist at the same time；